deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -376,7 +378,12 @@ Resources:
                 - aws s3 rm s3://$DEST_BUCKET --recursive
             build:
               commands:
-                - aws s3 cp --recursive . s3://$DEST_BUCKET/
+                - SOURCE_SHA=$(cat sha.txt)
+                - aws s3 cp --recursive . s3://$DEST_BUCKET/ --exclude "sha.txt"
+                - DATE=$(date)
+            post_build:
+              commands:
+                - curl -i -X POST -H "Authorization:token ${OAUTH_TOKEN}" "https://api.github.com/repos/ndlib/mellon-blueprints/statuses/${SOURCE_SHA}" -d "{\"state\":\"success\",\"description\":\"Deployed to production on ${DATE}\",\"context\":\"${CODEBUILD_INITIATOR}\",\"target_url\":\"https://url_to_production\"}"
```

Comment:
Rather than the mellon-blueprints repo, we should be tagging the repository that the CodePipeline is building and deploying. I believe in this case, this is ndlib/mellon-website rather than ndlib/mellon-blueprints.

We also will want to change "https://url-to-production" to the actual mellon URL, which is "https://mellon-unified.library.nd.edu"

Reply:
Added those changes